app/api/exports.py

```diff
@@ -39,9 +39,7 @@ def export_event(event_identifier):
         event_id = event_identifier
 
     if not has_access('is_coorganizer', event_id=event_id):
-        return abort(
-            assert_forbidden(argument=False)
-        )
+        return assert_forbidden()
```

Comment:
Where is the parameter passed into it? It'll return unconditionally.

My comment was a function which takes `has_access('is_coorganizer', event_id=event_id)` as an argument and raises forbidden error if it is false, so that it can be used for other checks as well